examples/5.deposit_to_margin_bank.ts

```diff
@@ -0,0 +1,31 @@
+/**
```

Comment:
I think these three examples can be merged into one
- deposit usdc to margin bank
- query usdc balance from margin bank
- withdraw usdc from margin bank